tensorflow/lite/micro/tools/project_generation/create_tflm_tree.py

```diff
@@ -97,18 +98,65 @@ def _create_tflm_tree(prefix_dir, makefile_options):
   _add_third_party_code(prefix_dir, makefile_options)
 
```

Comment:
Don't we need to optionally call _create_examples_tree() here?

Reply:
Good catch. The missing line (in the __main__ block) is from a bad merge. And the reason why the CI is failing. Fixed.